src/Controller/Api/AliasController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Controller\Api;
+
+use App\Dto\PasswordDto;
+use App\Dto\LocalpartDto;
+use App\Entity\Alias;
+use App\Handler\AliasHandler;
+use App\Handler\DeleteHandler;
+use App\Repository\AliasRepository;
+use App\Exception\ValidationException;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Bundle\SecurityBundle\Security;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\Routing\Attribute\Route;
+use Symfony\Component\PasswordHasher\Hasher\PasswordHasherFactoryInterface;
+use Symfony\Component\HttpKernel\Attribute\MapRequestPayload;
+
+class AliasController extends AbstractController
+{
+    private readonly AliasRepository $aliasRepository;
+
+    public function __construct(
+        private readonly Security $security,
+        private readonly AliasHandler $aliasHandler,
+        private readonly DeleteHandler $deleteHandler,
+        private readonly EntityManagerInterface $manager,
+    ) {
+        $this->aliasRepository = $manager->getRepository(Alias::class);
+    }
+
+    #[Route('/api/user/aliases', name: 'get_user_aliases', methods: ['GET'])]
+    public function getAliases(): JsonResponse
+    {
+        $user = $this->security->getUser();
```

Comment:
Doesn't it work to use `$this->getUser()` directly from the parent `AbstractController`?